airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -325,6 +325,13 @@
   documentationUrl: https://docs.airbyte.io/integrations/sources/db2
   icon: db2.svg
   sourceType: database
+- name: Infinicept
+  sourceDefinitionId: 11856c6d-37c3-4cdf-81c0-f1fe2fbaf280
+  dockerRepository: airbyte/source-infinicept
```

Comment:
Should this also point to our own docker repo?

Reply:
I think this should remain as is, since it will be referenced during deployment